src/main/java/org/truffleruby/core/string/StringNodes.java

```diff
@@ -4797,10 +4796,9 @@ protected RubyString stringFromByteArray(
 
     }
 
-    // TODO: rename
-    public abstract static class NewSingleByteOptimizableNode extends RubyBaseNode {
-        public static NewSingleByteOptimizableNode create() {
-            return NewSingleByteOptimizableNodeGen.create();
+    public abstract static class SingleByteOptimizableNode extends RubyBaseNode {
```

Comment:
Thanks. I meant to rename this after removing `RopeNodes.SingleByteOptimizableNode`.